src/Features/CSharp/Portable/CodeFixes/HideBase/HideBaseCodeFixProvider.cs

```diff
@@ -42,7 +51,25 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
                 return;
```

Comment:
same issue here.  but this seems to be a different feature.  so it's buggy that feature was written that way.